### PR TITLE
Change search bar

### PR DESCRIPTION
**Pull request by saloni-080601**
Updated search bar

### CODE REVIEW

File: src/components/SearchBar/SearchPopup.js [hunk 1]
```diff
@@ -47,21 +47,18 @@ function SearchPopup() {
...
 
   const handleSearchChange = (e) => {
-    if (!e.target.value) {
-      history.replace("");
-    } else {
+    if (e.key === "Enter") {
       history.replace(`/search-course/?search=${e.target.value}`);
+      setAnchorEl(null);
+      e.preventDefault();
+      setSearch(e.target.value);
```
Comment (Poonam-Singh-Bagh) [inline src/components/SearchBar/SearchPopup.js]: @saloni-080601 If you are updating the state on onChange then why are you updating here again?

**Comment by Poonam-Singh-Bagh** [comment]
image

@saloni-080601 fix the spacing

**Comment by Poonam-Singh-Bagh** [comment]
Screenshot from 2023-02-23 16-19-27

Showing 49 results, Whereas there is nothing

File: src/components/SearchBar/SearchPopup.js (hunk 1 above)
Comment (saloni-080601) [reply src/components/SearchBar/SearchPopup.js]: I forget to remove it from here

Comment (Poonam-Singh-Bagh) [reply src/components/SearchBar/SearchPopup.js]: Okay, then remove it from here

**Pull request merged**